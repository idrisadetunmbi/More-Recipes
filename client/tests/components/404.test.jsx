import React from 'react';
import renderer from 'react-test-renderer';

import NotFoundPage from '../../components/404';

describe('NotFoundPage component', () => {
  it('matches snapshot', () => {
    const component = renderer.create(<NotFoundPage />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
