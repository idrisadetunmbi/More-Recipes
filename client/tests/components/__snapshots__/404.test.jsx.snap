// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component matches snapshot 1`] = `
<div
  style={
    Object {
      "marginTop": "15rem",
      "p": Object {
        "fontFamily": "Open Sans",
      },
      "textAlign": "center",
    }
  }
>
  <h1>
    404
  </h1>
  <p
    style={
      Object {
        "fontFamily": "Open Sans",
      }
    }
  >
    This page does not exist
  </p>
</div>
`;
